image/default_image.go

```diff
@@ -170,50 +172,59 @@ func (d DefaultImageService) Login(RegistryURL, RegistryUsername, RegistryPasswd
 	return nil
 }
 
-func (d DefaultImageService) Delete(imageName string) error {
+func (d DefaultImageService) Delete(imageArg string) error {
 	var (
 		images        []*v1.Image
 		image         *v1.Image
 		imageTagCount int
 		imageID       string
 		imageStore    = d.imageStore
+		isName        = strings.Contains(imageArg, ":")
```

Comment:
Using this to judge a args is name or not is not very good.

You can just try to delete it if it is a name, then try to delete it as ID